lib/constants/theme/app_theme.dart

```diff
@@ -65,8 +65,8 @@ class AppTheme extends ThemeExtension<AppTheme> with _$AppThemeTailorMixin {
     currentIndicatorBackgroundColorWithDefaultArt: constants.Theme.defaultPrimaryColor,
     drawerMenuItemColor: const Color(0xff3d3e42),
     glowSplashColor: constants.Theme.lightThemeSplashColor,
-    glowSplashColorOnContrast: Colors.white.withOpacity(0.13),
-    sliderInactiveColor: Colors.black.withOpacity(0.2),
+    glowSplashColorOnContrast: Colors.white.withValues(alpha: 0.13),
+    sliderInactiveColor: Colors.black.withValues(alpha: 0.2),
```

Comment:
It's not an equal conversion

Reply:
Are you sure? What makes you say that? From what I can see:
* `withOpacity(x)` -> `withAlpha((255.0 * x).round())` -> `Color.fromARGB((255.0 * x).round(), ...)` -> `this._fromRGBOC(..., ((255.0 * x).round() & 0xff) / 255, colorSpace)` -> `a = ((255.0 * x).round() & 0xff) / 255` which is basically `a = x`
* `withValues(alpha: x)` -> `a = x`

Reply:
Yep, you are right